Doyeon-k-fanshawe/group2_bootcamp
# Removing image

## What does the PR do?
* It deletes a image

## How do you check this PR?
* Pull down the branch
*  See removed image

## Screenshots 
None.
